FeedAPIChallenge/Feed API/FeedImageJSONMapper.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  FeedImageJSONMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Константин Богданов on 04.06.2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedImageJSONMapper: FeedImageMapper {
+	private struct DecodableFeedImage: Decodable {
+		let image_id: UUID
+		let image_desc: String?
+		let image_loc: String?
+		let image_url: URL
+
+		func feedImage() -> FeedImage {
+			return .init(id: image_id,
+			             description: image_desc,
+			             location: image_loc,
+			             url: image_url)
+		}
+	}
+
+	private struct Response: Decodable {
+		let items: [DecodableFeedImage]
+	}
+
+	func map(data: Data) throws -> [FeedImage] {
```

Comment:
Following from my previous comment, the `map` function could be made `static` so there would be no need to create an instance of `FeedImageJSONMapper` to call it.
```suggestion
	static func map(data: Data) throws -> [FeedImage] {
```